### PR TITLE
Create Standby pool cli

### PR DESCRIPTION
**Pull request by fangru2023**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

`az standby-vm-pool create`
`az standby-vm-pool update`
`az standby-vm-pool show`
`az standby-vm-pool list`
`az standby-vm-pool delete`

`az standby-container-group-pool create`
`az standby-container-group-pool show`
`az standby-container-group-pool list`
`az standby-container-group-pool delete`

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by mimckitt** [comment]
@yonzhan @jsntcy any update on this? Anything you need from us?

**Comment by fangru2023** [comment]
> ## ⚠️ Suggestions
> ### Module: standbypool
> * Update version to `1.0.0b1` in setup.py
> 
> #### Notes
> * Stable/preview tag is inherited from last release. If needed, please add `stable`/`preview` label to modify it.
> * Major/minor/patch/pre increment of version number is calculated by pull request code changes automatically. If needed, please add `major`/`minor`/`patch`/`pre` label to adjust it.
> * For more info about extension versioning, please refer to [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

Done

**Comment by fangru2023** [comment]
@jsntcy , any feedback?

Need review @yonzhan, @zhoxing-ms , @kairu-ms
cc: @bxubxu , @mimckitt

**Comment by mimckitt** [comment]
@jsntcy,@yonzhan, @zhoxing-ms , @kairu-ms

I believe we are ready for review. Will need to get this merged this week so we can meet CLI may launch.

File: src/standbypool/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Azure CLI Standbypool Extension #
+This is an extension to Azure CLI to manage Standbypool resources.
+
+## How to use ##
+Please add commands usage here.
```
Comment (kairu-ms) [inline src/standbypool/README.md:5]: Please complete this readme file

File: src/service_name.json [hunk 2]
```diff
@@ -550,7 +550,17 @@
...
+    "AzureServiceName": "Standby Pool",
+    "URL": "https://aka.ms/StandbyPoolsForACI"
+  },
+  {
+    "Command": "az standby-vm-pool",
+    "AzureServiceName": "Standby Pool",
+    "URL": "https://aka.ms/StandbyPoolsForVMSS"
+  },
+  {
+    "Command": "az ",
```
Comment (kairu-ms) [inline src/service_name.json:563]:
```suggestion
    "Command": "az stack-hci-vm",
```

Comment (fangru2023) [reply src/service_name.json]: Thanks for pointing out, addressed.

File: src/standbypool/README.md (hunk 1 above)
Comment (fangru2023) [reply src/standbypool/README.md]: Thanks for pointing out, updated readme file.

**Comment by fangru2023** [comment]
Hi @kairu-ms, thanks for your reviewing, addressed your comments and publish a newer iteration, now it's ready to review again.

**Review by kairu-ms — APPROVED** [review]

**Comment by kairu-ms** [comment]
PR approved. @fangru2023, is it ready for the first release.

**Comment by mimckitt** [comment]
@kairu-ms I believe we are ready for merge. 

CC @fangru2023

**Comment by fangru2023** [comment]
> PR approved. @fangru2023, is it ready for the first release.

Yep, @kairu-ms, it's ready for the first release! Thanks for approval!

**Pull request merged**